src/pages/workspace/withPolicy.tsx

```diff
@@ -54,6 +53,34 @@ function getPolicyIDFromRoute(route: PolicyRoute): string {
     return route?.params?.policyID ?? '';
 }
 
+const taxPropTypes = PropTypes.shape({
+    /** Name of a tax */
+    name: PropTypes.string,
+
+    /** The value of a tax */
+    value: PropTypes.string,
+
+    /** Whether the tax is disabled */
+    isDisabled: PropTypes.bool,
+});
+
+const taxRatesPropTypes = PropTypes.shape({
+    /** Name of the tax */
+    name: PropTypes.string,
+
+    /** Default policy tax ID */
+    defaultExternalID: PropTypes.string,
+
+    /** Default value of taxes */
+    defaultValue: PropTypes.string,
+
+    /** Default foreign policy tax ID */
+    foreignTaxDefault: PropTypes.string,
+
+    /** List of tax names and values */
+    taxes: PropTypes.objectOf(taxPropTypes),
+});
```

Comment:
NAB but why those propTypes are still needed?

Reply:
`policyPropTypes` is used in `EditRequestPage.js`